framework/include/auxkernels/MortarNodalAuxKernel.h

```diff
@@ -61,6 +61,25 @@ class MortarNodalAuxKernelTempl : public AuxKernelTempl<ComputeValueType>,
 
   /// Old value
   const typename OutputTools<ComputeValueType>::VariableValue & _u_old;
+
+  /// The current test function index
+  unsigned int _i;
+
+  /// The current quadrature point index
+  unsigned int _qp;
+
+  /// The shape functions for the variable associated with the lower-dimensional element, e.g. the
+  /// shape functions for the nodes connected to the lower-dimensional element
+  const typename OutputTools<ComputeValueType>::VariableTestValue & _test_lower;
+
+  /// Member for handling change of coordinate systems (xyz, rz, spherical) on mortar elements
+  const MooseArray<Real> & _coord_msm;
+
+private:
+  // These are invalid for use because they are only meant for working with the higher-dimensional
+  // elements
```

Comment:
so you are changing the access qualifier here to prevent use by derived classes?